generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-fop/pom.xml

```diff
@@ -57,7 +57,7 @@
     <dependency>
       <groupId>org.apache.pdfbox</groupId>
       <artifactId>pdfbox</artifactId>
-      <version>${quarkus-qute-web.version}</version>
```

Comment:
This looks incorrect to me

Reply:
This should be fine: https://github.com/quarkusio/quarkus-platform/pull/1234#discussion_r1665536264